### PR TITLE
all: create SECURITY.md

### PR DESCRIPTION
**Pull request by katiehockman**

### CODE REVIEW

**Review by ajgajg1134 — APPROVED** [review]

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+Please see our [Support Policy](README.md#support-policy)
+
+## Reporting a Vulnerability
+
+If you have found a security issue, please contact the security team directly at security@datadoghq.com.
```
Comment (nsrip-dd) [inline SECURITY.md:9]: Does this apply specifically to vulnerabilities in dd-trace-go itself? As opposed to possible vulnerabilities in our dependencies?

Comment (ajgajg1134) [reply SECURITY.md:9]: OOooooo Very good point. We should clarify this is only for vulnerabilities in dd-trace-go itself NOT in dependencies

**Review by felixge — DISMISSED** [review]
LGTM (once the clarification around dependencies has been made)

File: SECURITY.md (hunk 1 above)
Comment (katiehockman) [reply SECURITY.md]: I went ahead and clarified that. Take another look 👍

**Review by ajgajg1134 — APPROVED** [review]

**Pull request merged**